### PR TITLE
Add support for UNWIND

### PR DESCRIPTION
**Pull request by Mats-SX**

### CODE REVIEW

File: caps-spark/src/main/scala/org/opencypher/caps/impl/spark/physical/operators/UnaryOperators.scala [hunk 1]
```diff
@@ -77,6 +80,54 @@ final case class Scan(in: PhysicalOperator, inGraph: LogicalGraph, v: Var, heade
...
+              records.data.crossJoin(df)
+
+            case x =>
+              Raise.invalidArgument("a list", x)
+          }
+
+          // the list lives in a column; we flatMap one output row for each element in the list
+        case expr =>
+          val listColIdx = records.data.schema.fieldIndex(SparkColumnName.of(expr))
+          val unwound = records.data.flatMap { row =>
```
[the remark this answers is not in the document]
Comment (DarthMax) [reply caps-spark/src/main/scala/org/opencypher/caps/impl/spark/physical/operators/UnaryOperators.scala]: https://spark.apache.org/docs/1.6.2/api/java/org/apache/spark/sql/functions.html#explode(org.apache.spark.sql.Column)

File: caps-core/src/main/scala/org/opencypher/caps/impl/flat/FlatOperatorProducer.scala [hunk 2]
```diff
@@ -140,6 +140,12 @@ class FlatOperatorProducer(implicit context: FlatPlannerContext) {
     Aggregate(aggregations, group, in, newHeader)
   }
 
+  def unwind(list: Expr, item: Var, in: FlatOperator): Unwind = {
+    val (header, _) = in.header.update(addContent(OpaqueField(item)))
```
[the remark this answers is not in the document]
Comment (Mats-SX) [reply caps-core/src/main/scala/org/opencypher/caps/impl/flat/FlatOperatorProducer.scala]: Yeah, that's a smell if I ever saw one :)

File: caps-spark/src/main/scala/org/opencypher/caps/impl/spark/physical/operators/UnaryOperators.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mats-SX) [reply caps-spark/src/main/scala/org/opencypher/caps/impl/spark/physical/operators/UnaryOperators.scala]: Interesting! I'll try it out.

**Comment by Mats-SX** [comment]
@DarthMax `explode` was the perfect fit, great find!

**Comment by Mats-SX** [comment]
Needs to wait for #242 before being merged

**Pull request merged**